ConsenSys/eventeum
# #124 Upgrading lombok and maven-failsafe to become OpenJDK-10+ compatible

## How to test
- Execute integration test suite:
```bash
$> mvn clean verify -DskipIntegrationTests=false
```

- Execute UnitTest and compile the package
```bash
$> mvn clean package
```

## Additional notes
Details of the machine used to launch the above steps:

```bash
uname -a
Linux ggarrido-laptop 4.15.0-76-generic #86-Ubuntu SMP Fri Jan 17 17:24:28 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

```
$> java --version
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1, mixed mode, sharing)
```

```
$> mvn --version
Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 11.0.6, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.15.0-76-generic", arch: "amd64", family: "unix"
```
